### PR TITLE
Merge update into main

### PR DESCRIPTION
**Pull request by FingerLeader**

### CODE REVIEW

**Comment by FingerLeader** [comment]
The CI fails because chaos mesh doesn't uses stress-ng to implement mem stress anymore, and incomming changes update the package `chaos-mesh`. These conditions cause CI to fail, so we need to upgrade mem attack first

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - release-*
+      - update
```
Comment (Andrewmatilde) [inline .github/workflows/ci.yml:7]: Plz delete this.

**Review by Andrewmatilde — APPROVED** [review]
Rest LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (FingerLeader) [reply .github/workflows/ci.yml]: Done it

Comment (Andrewmatilde) [reply .github/workflows/ci.yml]: LGTM

File: Makefile [hunk 2]
```diff
@@ -75,7 +75,12 @@ ifeq (,$(wildcard bin/tools/byteman))
 	tar zxvf ${BYTEMAN_DIR}.tar.gz
 	mv ${BYTEMAN_DIR} ./bin/tools/byteman
 endif
-
+ifeq (,$(wildcard bin/tools/memStress))
+	curl -fsSL -o memStress_v0.3-x86_64-linux-gnu.tar.gz https://github.com/chaos-mesh/memStress/releases/download/v0.3/memStress_v0.3-x86_64-linux-gnu.tar.gz
+	tar zxvf memStress_v0.3-x86_64-linux-gnu.tar.gz
+	mv memStress ./bin/tools/memStress
+endif
+curl -L https://github.com/chaos-mesh/memStress/releases/download/v0.3/memStress_v0.3-$NSEXEC_ARCH-linux-gnu.tar.gz
```
[the remark this answers is not in the document]
Comment (FingerLeader) [reply Makefile:83]: Oops! It is a mistake, I'll delete it

File: Makefile [hunk 3]
```diff
@@ -75,7 +75,12 @@ ifeq (,$(wildcard bin/tools/byteman))
 	tar zxvf ${BYTEMAN_DIR}.tar.gz
 	mv ${BYTEMAN_DIR} ./bin/tools/byteman
 endif
-
+ifeq (,$(wildcard bin/tools/memStress))
+	curl -fsSL -o memStress_v0.3-x86_64-linux-gnu.tar.gz https://github.com/chaos-mesh/memStress/releases/download/v0.3/memStress_v0.3-x86_64-linux-gnu.tar.gz
```
[the remark this answers is not in the document]
Comment (FingerLeader) [reply Makefile:79]: Cause others tools doesn't support multi platform, I think it' s meaningless to use `NSEXEC_ARCH` when downloading `memStress`

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**